### PR TITLE
feat(collector): apply collector schedule

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply collector schedule data from collector list API
- fixed collector schedule format
image

- will be applied 'in-progress' status after the job API checking

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 1]
```diff
@@ -201,27 +200,23 @@ const storeState = reactive({
...
-
-            if (closestHour) {
-                return state.current.set('h', closestHour).set('m', 0);
-            }
-            return state.current.set('h', sortedHours[0] ?? 0).set('m', 0).add(1, 'd');
-        }
-        return null;
+    scheduleActive: computed(() => {
+        const schedule = props.item.schedule;
+        return !!schedule;
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:205]: suggestion: How about `isScheduleActivated?`

```typescript
isScheduleActivated: computed<Boolean>(() => {
```

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue [hunk 2]
```diff
@@ -191,36 +190,30 @@ const props = withDefaults(defineProps<Props>(), {
...
-        const timeDiff = state.nextSchedule.diff(state.current, 'm');
-        return { diffHour: Math.floor(timeDiff / 60), diffMin: timeDiff % 60 };
+        if (props.item.schedule) {
+            const current = dayjs().utc();
+
+            const userCurrentTime = dayjs.tz(current, storeState.timezone);
+            const hours = props.item.schedule.hours ?? [];
+            const nextSchedule = hours.sort((a, b) => a - b).find((num) => num > userCurrentTime.hour());
+
+            const setNextSchedule = current.set('h', nextSchedule || 0).set('m', 0);
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue:212]: nitpick: this name feels like that it is a function. What about to change `nextSchedule` to 'nextScheduledHour' and `setNextSchedule` to `nextScheduledTime` ?

**Review by WANZARGEN — APPROVED** [review]
Looks good!
I left some comments!

**Pull request merged**